mapeditor/windownewmap.cpp

```diff
@@ -464,3 +503,23 @@ void WindowNewMap::on_cpuTeamsCombo_activated(int index)
 	updateTemplateList();
 }
 
+
+
+void WindowNewMap::on_mapOpt1_toggled(bool checked) 
```

Comment:
What is `mapOpt1` widget? Perhaps give it more clear name?

Reply:
Changed to sizeStandardRadio